### DISCUSSION TITLE
Sharing a chapter

### DISCUSSION THREAD

**Discussion by serrq** [PaulWoitaschek/Voice#1521] [Ideas]
Chapters for me are audio podcasts extracted from videos. So I have a need to share an episode with other people.

![Screenshot_2022-09-28-19-57-25-46_368b4f86ace9122103e958c9f267dd68 (1)](https://user-images.githubusercontent.com/113377431/192855289-de37faa2-a4e7-42ba-9f49-3c01d9abff71.jpg)

**Comment by prurigro** [comment]
To add to this, I've been using this app exclusively for podcasts for years now and it's amazing (thank you to everyone involved in building it!), I use podget to pull in new episodes on my desktop and then copy them to my phone when my queue is nearing the end.

I frequently find myself wanting to share an episode with someone though, and the podcast files typically have very random filenames (usually the podcast name + date + random string of text). I end up having to open the list of chapters, count how many came before the one I'm listening to, open the file manager, count the same number, open the next one in a different video playing app to verify it's the correct one, and share from there. If Voice had a share button it would save a ton of steps, and I can't think of a single feature I'd want from it after that :)
